src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    public String getDescription() {
+        return description;
+    }
+    @Override
+public String toString()
```

Comment:
Please consider cleaning up this indentation to adhere to the [java coding standards](https://se-education.org/guides/conventions/java/basic.html#layout). This applies to other parts of the code too